src/assets/scripts/aircraft/AircraftInstanceModel.js

```diff
@@ -2021,7 +2021,7 @@ export default class Aircraft {
             }
 
             // lock ILS if at the right angle and altitude
-            if ((abs(this.altitude - glideslope_altitude) < glideslope_window)
+            if (this.altitude < (glideslope_altitude + glideslope_window)
                 && (abs(offset_angle) < degreesToRadians(10))
                 && (offset[1] < ils)
```

Comment:
Agreed. I actually originally wanted to take it even further, and abstract it to the level of intercepting a lateral course, since this has countless applications (intercepting localizers, joining airways, VOR radials, holdings, even taxi paths, etc).